search_scale_for_roi_by_gps: 4.0
search_scale_for_roi_by_detection: 2.0
count_of_pictures_for_odometry: 2
low_pass_speed: 30
low_pass_coordinates: 50
angle_restriction_homography: 0.4
low_scale_restriction_homography: 0.5
high_scale_restriction_homography: 1.5

image_size_sift: 250
points_quality_sift: 0.9
nOctaveLayers_sift: 5
contrastThreshold_sift: 0.1
edgeThreshold_sift: 40
sigma_sift: 1.6

camera_pitch_angle: 0.2
use_imu: True
use_gps: True

publish_roi_img: True
publish_keypoints_matches_img: True
publish_between_img: True
publish_calculated_pose_img: True
